src/main/java/com/gradle/CustomBuildScanEnhancements.java

```diff
@@ -306,6 +306,10 @@ public void execute(BuildResultAdapter buildResult) {
                     addCustomValueAndSearchLink(develocity, "CI run", value));
                 envVariable("GITHUB_HEAD_REF", providers).filter(value -> !value.isEmpty()).ifPresent(value ->
                     buildScan.value("PR branch", value));
+                envVariable("GITHUB_ACTION", providers).ifPresent(value ->
+                        addCustomValueAndSearchLink(develocity, "CI step id", value));
```

Comment:
Why do we capture `GITHUB_ACTION` then call it `CI step id`? I think `CI action` would be more appropriate.